guides/developer/provider-setups/aws-dev-setup.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: single
+title: "Getting Set Up for Spinnaker Development on AWS"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+This guide describes how a developer can set up a [Local Git deployment](/setup/install/environment/#local-git) of Spinnaker on Amazon EC2.
+
+In this setup, Spinnaker runs on an EC2 instance. Code is edited on the laptop and then synced to the EC2 instance for testing.
```

Comment:
"...on the laptop..."

Or a desktop, right? Maybe just "local machine" or something like that?